### ISSUE TITLE
[com.google.fonts/check/old_ttfautohint] will fail for Variable Fonts

### ISSUE THREAD

**Issue by kontur** [googlefonts/fontbakery#2790]
## Observed behaviour

Hinting a Variable Font with `ttfautohint-vf` (as per [guidelines](https://gist.github.com/m4rc1e/8f4c4498519e8a36cd54e16a004275cb)) will result in an parse error due to the "outdated" and "non-release" version number:

> * FAIL: Failed to parse ttfautohint version values: installed = '1.8.3'; used_in_font = '1.8.1.43-b0c9' [code: parse-error]

## Expected behaviour

Pass

## Resources and exact process needed to replicate

Install `ttfautohint-vf` as [per instruction](https://gist.github.com/m4rc1e/8f4c4498519e8a36cd54e16a004275cb), run `ttfautohint-vf AnyVariableFont.ttf`, run `fontbakery check-googlefonts AnyVariableFont.ttf`

**Comment by m4rc1e** [comment]
Thanks for raising this.

We should probably allow the wip version of ttfautohint-vf to pass this test. I'm still unsure about hinting in general but we can add this for the time being.

**Comment by kontur** [comment]
Roger that. @m4rc1e I also inquired form Werner Lemberg, who referred me forward to Ewald Hew, the maintainer/developer of the [-vf fork](https://gitlab.com/ewaldhew/tah-vf), and there seems to be a lot more new work since that 1.8.1 binary. C -noob that I am I failed to build a newer binary, but I'd be curious as to how the versions differ — and in that respect the ttfautohint version allowed might not just be that one particular binary.

Maybe the particular version check for ttfautohint should be disabled for _variable fonts_ altogether.
